app/components/validators.js

```diff
@@ -15,9 +17,7 @@ export const required = val => {
   }
 
   if (val !== null && typeof val === 'object') {
-    return values(val).filter(function(v) {
-      return v
-    }).length
+    return values(val).filter(v => v).length
```

Comment:
not related to your change but I think you could remove `.filter(v => v)` as it's returning 1 for 1